### PR TITLE
Feat/analytics

### PR DESCRIPTION
**Pull request by olastor**

### CODE REVIEW

File: src/app/routes/_analytics.router.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        data: [],
+        labels: []
+      };
+
+      let dtEnd = Date.now();
+      let dtStart = dtEnd - interval;
+
+      while (dtStart > minDate) {
+        // Use MongoDB's id to find new users within the current timespan
+        const minObjID = ObjectId(Math.floor(dtStart / 1000).toString(16) + "0000000000000000");
```
Comment (binarious) [inline src/app/routes/_analytics.router.js]: Could you please explain this a bit in detail?

**Review by binarious — CHANGES_REQUESTED** [review]
Please add basic tests for the analytics routes (you can copy existing ones) and fix the linting issues.

File: src/app/routes/_analytics.router.js (hunk 1 above)
Comment (olastor) [reply src/app/routes/_analytics.router.js]: http://stackoverflow.com/questions/6452021/getting-timestamp-from-mongodb-id The first 4 Bytes of a MongoDB id contain a timestamp of the creation. That means when an object id is greater than another one, the creation datetime of the first one has to be after (or at the same time) the creation of the second one.

Comment (binarious) [reply src/app/routes/_analytics.router.js]: Thank you for the clarification.

**Review by binarious — APPROVED** [review]

**Pull request merged**